object.cpp

```diff
@@ -609,27 +609,41 @@ int Object::FleetLoad()
 int Object::SailThroughCheck(int dir)
 {
 	if (IsFleet()) {
+		// if target region doesn't exist, cannot be sailed into
+		if (!region->neighbors[dir]) {
```

Comment:
more common style in code is not use brackets for single line if like

`if (!region->neighbors[dir]) return 0;`

I am not a big fan of it but to keep code consistent...